ext/hal/nxp/mcux/devices/LPC55S69/fsl_powerquad_basic.c

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2018 NXP
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include "fsl_powerquad.h"
```

Comment:
powerquad should go in the shared drivers folder instead

Reply:
Thank you @MaureenHelm powerquad was filtered as part of fsl_power by the script. Waiting for a review from @dbkinder and @MeganHansen for another force push.